<?php defined('BASEPATH') OR exit('No direct script access allowed'); ?>

ERROR - 2019-05-16 11:29:13 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:29:13 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:29:49 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:29:49 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:30:11 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:30:11 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:32:26 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:32:26 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:32:29 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:32:29 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:32:54 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:32:54 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:35:56 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:35:56 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:35:57 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:35:57 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:35:58 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:35:58 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:36:39 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:36:39 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:37:13 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:37:13 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:37:15 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:37:15 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:37:15 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:37:15 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:37:15 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:37:15 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:37:15 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:37:15 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:37:41 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:37:41 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:41:57 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:41:57 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:43:31 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:43:31 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:45:13 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:45:18 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:45:51 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:45:52 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:45:52 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:45:52 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:45:52 --> 404 Page Not Found: Product/index
ERROR - 2019-05-16 11:47:12 --> Severity: Notice --> Undefined property: Product::$Main_model C:\xampp\htdocs\shopify\monika\application\controllers\Product.php 19
ERROR - 2019-05-16 11:47:12 --> Severity: error --> Exception: Call to a member function getUserDetails() on null C:\xampp\htdocs\shopify\monika\application\controllers\Product.php 19
ERROR - 2019-05-16 11:47:12 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:48:02 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:48:02 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:48:16 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:48:16 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:48:17 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:48:17 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:48:17 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:48:17 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:48:26 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:48:26 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:49:46 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:49:46 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:50:17 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:50:17 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:50:18 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:50:18 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:50:18 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:50:18 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:51:02 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:51:02 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:51:02 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:51:02 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:51:03 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:51:03 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:51:39 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:51:39 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:52:29 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:52:29 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:52:29 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:52:29 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:52:29 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:52:29 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:52:29 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:52:29 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:52:30 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:52:30 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:52:30 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:52:30 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:55:11 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:55:11 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:55:12 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:55:12 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:55:13 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:55:13 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 11:55:13 --> Severity: error --> Exception: syntax error, unexpected '[', expecting ')' C:\xampp\htdocs\shopify\monika\application\models\Main_Model.php 25
ERROR - 2019-05-16 11:55:13 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of ParseError given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(ParseError))
#1 [internal function]: _exception_handler(Object(ParseError))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 12:06:50 --> 404 Page Not Found: Users/index
ERROR - 2019-05-16 12:11:53 --> Severity: Notice --> Undefined variable: productData C:\xampp\htdocs\shopify\monika\application\views\product_view.php 25
ERROR - 2019-05-16 12:11:53 --> Severity: Warning --> Invalid argument supplied for foreach() C:\xampp\htdocs\shopify\monika\application\views\product_view.php 25
ERROR - 2019-05-16 12:12:41 --> Severity: error --> Exception: Cannot use object of type mysqli as array C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:12:41 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 12:14:46 --> Severity: error --> Exception: Cannot use object of type mysqli as array C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:14:46 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 12:14:53 --> 404 Page Not Found: Product/exportocvs
ERROR - 2019-05-16 12:14:56 --> Severity: error --> Exception: Cannot use object of type mysqli as array C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:14:56 --> Severity: Error --> Uncaught TypeError: Argument 1 passed to CI_Exceptions::show_exception() must be an instance of Exception, instance of Error given, called in C:\xampp\htdocs\shopify\monika\system\core\Common.php on line 662 and defined in C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php:190
Stack trace:
#0 C:\xampp\htdocs\shopify\monika\system\core\Common.php(662): CI_Exceptions->show_exception(Object(Error))
#1 [internal function]: _exception_handler(Object(Error))
#2 {main}
  thrown C:\xampp\htdocs\shopify\monika\system\core\Exceptions.php 190
ERROR - 2019-05-16 12:16:19 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:19 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:19 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:19 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: productid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 27
ERROR - 2019-05-16 12:16:20 --> Severity: Notice --> Undefined index: roomleid C:\xampp\htdocs\shopify\monika\application\views\product_view.php 28
ERROR - 2019-05-16 12:22:06 --> Query error: Unknown column 'roomle' in 'field list' - Invalid query: SELECT `productid`, `roomle`
FROM `product`
ERROR - 2019-05-16 12:30:01 --> 404 Page Not Found: Users/index
ERROR - 2019-05-16 12:30:09 --> 404 Page Not Found: Products/index
ERROR - 2019-05-16 12:30:11 --> 404 Page Not Found: Products/index
ERROR - 2019-05-16 12:30:47 --> 404 Page Not Found: Users/exportocvs
ERROR - 2019-05-16 14:51:32 --> Severity: error --> Exception: Unable to locate the model you have specified: Csv_import_ci_model C:\xampp\htdocs\shopify\monika\system\core\Loader.php 314
ERROR - 2019-05-16 14:53:24 --> Unable to load the requested class: Csvimport
ERROR - 2019-05-16 14:55:22 --> Unable to load the requested class: Csvimport
ERROR - 2019-05-16 14:55:30 --> Unable to load the requested class: Csvimport
ERROR - 2019-05-16 15:00:26 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:00:26 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:00:26 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:00:33 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:00:33 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:00:33 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:00:49 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:00:49 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:00:49 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:17:54 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:17:54 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:17:54 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:23:13 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:23:13 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:23:13 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:23:20 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:23:20 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:23:20 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:26:35 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:26:35 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:26:35 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:26:35 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:26:36 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:26:36 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:26:36 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:26:44 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:26:44 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:26:44 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:26:44 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:33:16 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:33:16 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:33:16 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:33:21 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:33:21 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:33:21 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:41:03 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:41:03 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:41:03 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:43:12 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:43:12 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:43:12 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:43:12 --> 404 Page Not Found: Js/bootstrap.min.js
ERROR - 2019-05-16 15:43:31 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:43:31 --> 404 Page Not Found: Jqueryminjs/index
ERROR - 2019-05-16 15:43:31 --> 404 Page Not Found: Bootstrapmincss/index
ERROR - 2019-05-16 15:43:31 --> 404 Page Not Found: Js/bootstrap.min.js
